bundles/FileExplorerBundle/src/Installer.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Pimcore\Bundle\FileExplorerBundle;
+
+use Pimcore\Extension\Bundle\Installer\SettingsStoreAwareInstaller;
+
+class Installer extends SettingsStoreAwareInstaller
```

Comment:
The installer must be defined in the services.yaml https://github.com/pimcore/pimcore/blob/11.x/bundles/GlossaryBundle/config/services.yaml#L10